src/infoview.ts

```diff
@@ -36,26 +38,34 @@ class InfoDocument implements Disposable {
     }
 
     get text() {
-        return this.lines.join('\n');
+        return this.contents;
     }
 
     dispose() {
         for (const s of this.subscriptions) s.dispose();
     }
 
     private render() {
-        this.lines = [];
-        this.renderHeader();
-        this.renderGoal();
-        this.renderMessages();
+        this.contents = this.renderHeader() + this.renderGoal() + this.renderMessages();
+    }
+
+    private stylesheet() {
+        return `#settings { display: block; text-align: right; }
+            h1 {
+                display: block;
+                background-color: blue;
```

Comment:
The split view sounds reasonable.

Re: color themes.  I absolutely agree that we should try to fit in with the vscode theme.  Unfortunately, I don't think there is any way to obtain theme colors from an extension.  The HTML preview gets injected with a few values, but that's just [foreground/background colors, and fonts](https://github.com/Microsoft/vscode/blob/3b8a3ca25e5ed631c6a786792ac85cc817b09891/src/vs/workbench/parts/html/browser/webview.ts#L182-L186).

I think the best that we can do is pick colors that fit with the default Dark+ and Light+ themes.  The info view should really look like a part of vscode, and not call attention to itself.